packages/flutter/test/rendering/paragraph_test.dart

```diff
@@ -36,6 +36,19 @@ void main() {
     expect(offset50.dy, greaterThan(offset5.dy));
   });
 
+  test('getFullHeightForCaret control test', () {
+    final RenderParagraph paragraph = RenderParagraph(
+      const TextSpan(text: _kText,style: TextStyle(fontSize: 10.0)),
+      textDirection: TextDirection.ltr,
+    );
+    layout(paragraph);
+
+    const Rect caret = Rect.fromLTWH(0.0, 0.0, 2.0, 20.0);
+
+    final double height5 = paragraph.getFullHeightForCaret(const TextPosition(offset: 5), caret);
+    expect(height5, equals(10.0));
```

Comment:
Per @Hixie suggestion I updated this test to be more strict in comparing expected value.